agent/taskresource/credentialspec/credentialspec_windows.go

```diff
@@ -529,9 +529,15 @@ func (cs *CredentialSpecResource) clearCredentialSpec() {
 			seelog.Debugf("Skipping cleanup of local credentialspec file option: %s", key)
 			continue
 		}
-		credSpecSplit := strings.SplitAfterN(value, "credentialspec=", 2)
+		// Split credentialspec to obtain local file-name
+		credSpecSplit := strings.SplitAfterN(value, "credentialspec=file://", 2)
```

Comment:
What will this do in the s3 or ssm cases?

Reply:
We do not want to cleanup the credentialspec's specified to use a local-file. 
However for S3/SSM options, we fetch the remote artifact and map it to a local file which is then passed onto the actual container. When the task is stopped and cleanup is triggered; this would cleanup the local artifact which was fetched from S3/SSM.